### ISSUE TITLE
[FEATURE] Homepage to display application statistics per day

### ISSUE THREAD

**Issue by eddiejaoude** [EddieHubCommunity/LinkFree#2118]
### Description

Display 3 stats on homepage per day

- users
- profile views
- link clicks

The data is stored here https://github.com/EddieHubCommunity/LinkFree/blob/main/models/Stats.js

**Comment by vijay-jaisankar** [comment]
https://d3-graph-gallery.com/ and https://www.chartjs.org/ are  great graphing libraries for this purpose

**Comment by thevinitgupta** [comment]
Is this data being stored anywhere?

**Comment by thevinitgupta** [comment]
Has this feature already been implemented?

**Comment by eddiejaoude** [comment]
No the issue is still open, let us know any ideas you have before starting work on it

**Comment by thevinitgupta** [comment]
![image](https://user-images.githubusercontent.com/65801700/210966092-eda3cdf3-8ac8-4ac5-96b7-790b26299c8a.png)

Isn't this the feature that was mentioned?

**Comment by eddiejaoude** [comment]
OMG You are correct, I was confusing this issue with another issue https://github.com/EddieHubCommunity/LinkFree/issues/2587

Thank you @thevinitgupta 👍
